src/device-registry/config/kafkajs.js

```diff
@@ -28,8 +29,21 @@ const runKafkaConsumer = async () => {
   logText("connecting to kafka consumer...");
   try {
     await kafkaConsumer.connect();
+    await kafkaConsumer.subscribe({
+        topic: constants.HOURLY_MEASUREMENTS_TOPIC,
```

Comment:
Brilliant! This is a reasonable approach as we continue to iterate around this topic! Thanks @BenjaminSsempala !